src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -39,24 +43,59 @@ public static void Initialize(this AView platformView, IView view)
 				visibility = (int)view.Visibility.ToPlatformVisibility();
 			}
 
-			// NOTE: use named arguments for clarity
-			PlatformInterop.Set(platformView,
-				visibility: visibility,
-				layoutDirection: (int)GetLayoutDirection(view),
-				minimumHeight: (int)platformView.ToPixels(view.MinimumHeight),
-				minimumWidth: (int)platformView.ToPixels(view.MinimumWidth),
-				enabled: view.IsEnabled,
-				alpha: (float)view.Opacity,
-				translationX: platformView.ToPixels(view.TranslationX),
-				translationY: platformView.ToPixels(view.TranslationY),
-				scaleX: (float)(view.Scale * view.ScaleX),
-				scaleY: (float)(view.Scale * view.ScaleY),
-				rotation: (float)view.Rotation,
-				rotationX: (float)view.RotationX,
-				rotationY: (float)view.RotationY,
-				pivotX: pivotX,
-				pivotY: pivotY
-			);
+			if (wrapperView is not null)
+			{
+				// Apply transform properties to the wrapper
+				wrapperView.TranslationX = wrapperView.ToPixels(view.TranslationX);
+				wrapperView.TranslationY = wrapperView.ToPixels(view.TranslationY);
+				wrapperView.ScaleX = (float)(view.Scale * view.ScaleX);
+				wrapperView.ScaleY = (float)(view.Scale * view.ScaleY);
+				wrapperView.Rotation = (float)view.Rotation;
+				wrapperView.RotationX = (float)view.RotationX;
+				wrapperView.RotationY = (float)view.RotationY;
+				wrapperView.PivotX = pivotX;
+				wrapperView.PivotY = pivotY;
+
+				// Apply non-transform properties to the inner view
+				PlatformInterop.Set(platformView,
```

Comment:
This is duplicated two times, can create a method with all the transformation properties as parameters?

Reply:
I have made the changes. please let me know if you have any other concerns. @jsuarezruiz